src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Config/ServiceBusOptions.cs

```diff
@@ -27,6 +27,8 @@ public ServiceBusOptions()
             {
                 MaxConcurrentCalls = 16
             };
+
+            SessionHandlerOptions = new SessionHandlerOptions(ExceptionReceivedHandler);
```

Comment:
When we create MessageHandler options we default MaxConcurrent to 16 because the default is 1. Do we need to do the same for SessionHandlerOptions.MaxConcurrentSessions? What is the default there? If it is 1, the behavior won't be what users expect likely.

Reply:
`SessionHandlerOptions.MaxConcurrentSessions` is defaulted to 16. We do not need to do something here.